pygmt/src/blockm.py

```diff
@@ -73,6 +134,35 @@ def blockmedian(table, outfile=None, **kwargs):
         - None if ``outfile`` is set (filtered output will be stored in file
           set by ``outfile``)
     """
+    return _blockm(block_method="blockmedian", table=table, outfile=outfile, **kwargs)
+
+
+def _blockm(block_method, table, outfile, **kwargs):
```

Comment:
More just a matter of style, but I feel like this `_blockm` function should be placed near the top of the file (before the actual `blockmean` and `blockmedian` functions).

Reply:
OK, done in https://github.com/GenericMappingTools/pygmt/pull/1092/commits/f34e77842ab870e3124cf8b15df8a1080ab946f5